### ISSUE TITLE
Possible to automatically exclude/mark already discovered?

### ISSUE THREAD

**Issue by Svish** [xanthics/gw2craft#51]
Similar to how this is able to fetch what I have of crafting materials in the bank, would it be possible to also fetch which recipes I already have? And then either just mark those for me, or even better, calculate a plan without those, and with my current level(#32) as a starting point?

**Comment by xanthics** [comment]
Guides are created and pushed to the site hourly.  There is no back-end on the webpages so custom guides, based on level and known discoveries, aren't possible.  Some type of alert that you have already discovered a recipe used in the guide may be possible though.

For marking off purchased recipes, that is possible.  However the account/recipes endpoint requires "unlocks" instead of "inventories" permissions, contrary to the hints for the permissions.  I've raised that as a bug in the forums and will wait to hear back before making any changes.  [Reference](https://en-forum.guildwars2.com/discussion/121205/bug-with-accounts-recipes-permission-requires-wrong-permission/p1?new=1)

**Comment by Svish** [comment]
Yeah, if all it did was to put a checkmark next to what I've already done, that would be very helpful 😊👍

And giving another permission on the api key is not an issue for me at least 🙂
